internal/backend/schedule/schedule.go

```diff
@@ -0,0 +1,98 @@
+package schedule
+
+import (
+	"context"
+	"fmt"
+
+	"go.temporal.io/sdk/client"
+	"go.uber.org/zap"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/dispatcher"
+	"go.autokitteh.dev/autokitteh/internal/backend/temporalclient"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+type temporalScheduleImpl struct {
+	z      *zap.Logger
+	tmprl  temporalclient.Client
+	events sdkservices.Events
+}
+
+type TemporalSchedule interface {
```

Comment:
```suggestion
type Schedule interface {
```

and elsewhere. the fact it's temporal is an impl detail.

Reply:
done